### PR TITLE
feat: Add Docker cheatsheets (english & spanish)

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s)

### Description

Added a category and list  the most ranked cheatsheets in it.

### Why is this valuable (or not)?

Cheatsheets help to remember the most important commands.

Here some of related with Docker CLI, Dockerfiles and `docker-compose.yml`

There are also someones with best practices about.

### How do we know it's really free?
Unlicensed, CC3-By or MIT projects

### For book lists, is it a book? For course lists, is it a course? etc.
Its genre is cheatsheets

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 
 * [CI/CD for Monorepos: Effectively building, testing, and deploying code with monorepos](https://github.com/semaphoreci/book-monorepo-cicd) - Pablo Tomas Fernandez Zavalia, Marko Anastasov, SemaphoreCI (PDF, EPUB, Kindle)
 * [CI/CD with Docker and Kubernetes Book](https://github.com/semaphoreci/book-cicd-docker-kubernetes) - Marko Anastasov, Jérôme Petazzoni, Pablo Tom F. Zavalia, SemaphoreCI (PDF, EPUB, Kindle)
+* [Docker Cheat Sheet](https://github.com/wsargent/docker-cheat-sheet) - Will Sargent, et al.
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:228]: It's a cheat sheet.

**Review by eshellman — CHANGES_REQUESTED** [review]
it's a lot of cheat sheets!

File: books/free-programming-books-subjects.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-subjects.md:228]: > it's a lot of cheat sheets!

Too much?

Comment (eshellman) [reply books/free-programming-books-subjects.md:228]: can save pruning/glossing for later. Are any of them older? if so, maybe year would be useful?

Comment (davorpa) [reply books/free-programming-books-subjects.md:228]: I'll check it :+1: again

Comment (davorpa) [reply books/free-programming-books-subjects.md]: > It's a cheat sheet.

Resolved at 82457bc55591c61ff60afff83c302c5a4be7be87

**Comment by davorpa** [comment]
### NOTE:

Needs help with discrimination about what resources are really helpful

**Comment by eshellman** [comment]
somehow the changes to .gitignore got included in this PR

**Comment by davorpa** [comment]
> somehow the changes to .gitignore got included in this PR

Due to a cherry pick in cf4a06ab32b858984717906fe73ead4f6d496442  instead of full merge of it source (#6674 : b3f1b8e77229c15982416e4aa93d75ecaaaecabe) . It doesn't conflict because is the same content, I thought

**Comment by eshellman** [comment]
try an update from main

**Comment by davorpa** [comment]
> try an update from main

Clean merge Done! 👍

**Pull request merged**